README.md

```diff
@@ -106,6 +106,7 @@ These are disabled by default, and can be enabled with the `/github subscribe ow
 - `reviews` - Pull request reviews
 - `comments` - New comments on issues and pull requests
 - `branches` - Created or deleted branches
+- `branches:tag` - Created tags
```

Comment:
From an implementation perspective I think it makes sense to include this on `branches`, but from a user perspective, would make sense to include this as a top-level setting? `branches:tag` feels weird, whereas `/github subscribe owner/repo tags` makes a little more sense to me.